marketpulse/main/migrations/0009_auto_20150224_1356.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+CARRIERS = {'Telefonica Digital' : [{'Movistar': ['Spain', 'Colombia', 'Venezuela', 'Peru',
+                                                  'Uruguay', 'Mexico', 'Chile', 'El Salvador',
+                                                  'Nicaragua', 'Guatemala', 'Costa Rica']},
+                                    {'Vivo': ['Brazil']},
+                                    {'O2': ['Germany']}],
+            'Deutsche Telekom': [{'T-Mobile': ['Poland', 'Czech Republic', 'Macedonia']},
+                                 {'Congstar': ['Germany']},
+                                 {'Telekom': ['Hungary']},
+                                 {'Cosmote': ['Greece']}],
+            'Telenor': [{'Telenor': ['Hugnary', 'Serbia', 'Montenegro']},
+                        {'Grameenphone': ['Bangladesh']}],
+            'Telecom Italia': [{'TIM': ['Italy']}],
+            'America Movil': [{'Telcel': ['Mexico']}],
+            'Retailer': [{'E.Leclerc': ['France']}],
+            'Snapdeal': [{'Spice': ['India']}],
+            'JB Hifi': [{'ZTE': ['Australia']}],
+            'Alcatel OneTouch': [{'TCL': ['India']}],
+            'www.flicpart.com': [{'ZEN Mobil': ['India']}],
+            'Megafon': [{'Megafon': ['Russia']}],
+            'Cherry Mobile': [{'Cherry Mobile': ['Philippines']}],
+            'Banglalink': [{'Banglalink': ['Bangladesh']}],
+            'KDDI': [{'KDDI': ['Japan']}]
+            }
+
+
+def remove_carriers(apps, schema_editor):
+    Carrier = apps.get_model('main', 'Carrier')
+    Carrier.objects.all().delete()
+
+
+def add_carriers(apps, schema_editor):
+    from django_countries import countries
+
+    Carrier = apps.get_model('main', 'Carrier')
+    for parent_operator, carriers in CARRIERS.items():
```

Comment:
I don't really like the nested ``for`` but since this is a migration we can keep this.